charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-snapshot-validation-webhook-vpa.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: autoscaling.k8s.io/v1beta2
+kind: VerticalPodAutoscaler
+metadata:
+  name: csi-snapshot-webhook-vpa
+  namespace: {{ .Release.Namespace }}
+spec:
+  resourcePolicy:
+    containerPolicies:
+    - containerName: csi-snapshot-validation
+      minAllowed:
+        memory: 25Mi
```

Comment:
What about some CPU min allowed value?

Reply:
For now I am following the same behaviour as snapshot-controller.